core-bundle/src/Resources/contao/languages/pl/tl_image_size.xlf

```diff
@@ -3,7 +3,7 @@
     <body>
       <trans-unit id="tl_image_size.name.0">
         <source>Name</source>
-        <target>Imię i nazwisko</target>
+        <target>Nazwa wyświetlana</target>
```

Comment:
Please change to just `Nazwa`.

Reply:
I found it in newest translation from Contao 4.10, that's why I put it in my correction.